index.html

```diff
@@ -265,43 +263,213 @@ <h2 class="h-cust-head">Trusted by Thousands of Happy Customer</h2>
                 </div>
-                <p>
+                <p class="customer-response">
                   “Wow... I am very happy to use this VPN, it turned out to be
                   more than my expectations and so far there have been no
                   problems. LaslesVPN always the best”.
                 </p>
               </li>
               <li class="list-item-cust-boxes">
-                <img src="" alt="" />
-                <div>
-                  <span>Yessica Christy</span>
-                  <span>Shanxi, China</span>
+                <div class="customer-information">
+                  <img
+                    src="./images/Christy-img.png"
+                    alt="Profile photo of Christy's client"
+                  />
+                  <div class="customer-info-item">
+                    <h3 class="name">Yessica Christy</h3>
+                    <span class="place-of-residence">Shanxi, China</span>
+                  </div>
+                  <span class="rating">4.5</span>
                 </div>
-                <p>
+                <p class="customer-response">
                   “I like it because I like to travel far and still can connect
                   with high speed.”.
                 </p>
               </li>
               <li class="list-item-cust-boxes">
-                <img src="" alt="" />
-                <div>
-                  <span>Kim Young Jou</span>
-                  <span>Seoul, South Korea</span>
+                <div class="customer-information">
+                  <img
+                    src="./images/Kim-img.png"
+                    alt="Profile photo of Kim's client"
+                  />
+                  <div class="customer-info-item">
+                    <h3 class="name">Kim Young Jou</h3>
+                    <span class="place-of-residence">Seoul, South Korea</span>
+                  </div>
+                  <span class="rating">4.5</span>
                 </div>
-                <p>
+                <p class="customer-response">
                   “This is very unusual for my business that currently requires
                   a virtual private network that has high security.”.
                 </p>
               </li>
             </ul>
           </div>
+          <div class="btns-wrap">
+            <div class="left-bullets">
+              <div class="first-left-blt"></div>
+              <div class="grey-blts"></div>
+              <div class="grey-blts"></div>
+              <div class="grey-blts"></div>
+            </div>
+            <div class="right-btns">
+              <button
+                class="btn-with-arrow"
+                aria-label="click on the button with left arrow"
+              >
+                <span class="icon-left_arrow_icon_for_button arrow-size"></span>
+              </button>
+              <button
+                class="btn-with-arrow"
+                aria-label="click on the button with right arrow"
```

Comment:
With aria-label we should describe functionality of the element, but not the appearance. Maybe it will be better "go to the next slide"

Reply:
I agree with you, it will be better.